mmv1/third_party/terraform/go.mod

```diff
@@ -32,18 +32,18 @@ require (
 	github.com/stretchr/testify v1.10.0
 	go4.org/netipx v0.0.0-20231129151722-fdeea329fbba
 	golang.org/x/exp v0.0.0-20240719175910-8a7402abbf56
-	golang.org/x/net v0.41.0
+	golang.org/x/net v0.42.0
 	golang.org/x/oauth2 v0.30.0
-	google.golang.org/api v0.242.0
```

Comment:
could you provide some documentation that talks about needing to update the `go.mod` of terraform due to the compute class?

Reply:
This is basically the first version that contains this field in the API: https://github.com/googleapis/google-api-go-client/compare/v0.244.0...v0.245.0#diff-153e46830575ba652ce82bb20bd2271e1ae69560091fc3f0424687920aff8dfeR3805. Earlier versions do not have it

Reply:
Sure, created with API update https://github.com/GoogleCloudPlatform/magic-modules/pull/14848